test/common.py

```diff
@@ -2461,13 +2467,77 @@ def init_worker(counter, lock):
     counter.value += 1
 
 
+def list_processes_by_name(exe_name):
+  try:
+    import psutil
```

Comment:
You can just add this to `requirements-dev.txt` and import it unconditionally at the top.

Reply:
I don't think we require all end users to have python `psutil` module installed? Are we ok to require that unconditionally?

Reply:
it not end users, its just emscripten developers (that was the `-dev` part means).  We should have a relatively low bar for adding stuff to `requirements-dev.txt` because it doesn't effect our actual users.

Reply:
For example, we have sphinx in there, but we don't expect emscripten users to install sphinx.